Zend/tests/bug79919.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Bug #79919 (Stack use-after-scope in define)
+--SKIPIF--
+<?php
+if (PHP_OS_FAMILY === 'Windows') die('skip need ASan instrumentation');
```

Comment:
Might as well let it run. We don't typically skip tests that only fail under asan/valgrind.

Reply:
Ah, fine, even better, since the skip condition isn't correct anyway (clang on Windows would detect the issue).